utils.py

```diff
@@ -157,3 +159,22 @@ def balance(it):
 
                 last_label = e[1]
                 yield e
+
+
+def download(url, filename, block='█'):
+    with open(filename, 'wb') as file:
```

Comment:
Can you use a different name for the `file` variable? It's a Python keyword, so it gets highlighted differently than variables.